docs/quickstart-azure.md

```diff
@@ -249,7 +249,7 @@ As worker nodes are managed by machine-controller, they can be scaled up and dow
 (including to 0) using Kubernetes API.
 
 ```bash
-kubectl --namespace kube-system scale machinedeployment/pool1-deployment --replicas=3
+kubectl --namespace kube-system scale machinedeployment/<CLUSTER-NAME>-pool1 --replicas=3
```

Comment:
I think it's safe to port this trick with `kubectl get machinedeployments -n kube-system` to other quick-starts

Reply:
Sounds reasonable and is more robust against changes in how the name is generated. But will drop how it is generated for the same reason.